src/bindings/jna/CMakeLists.txt

```diff
@@ -27,6 +27,21 @@ if (Java_VERSION_MAJOR GREATER 0 AND Java_VERSION_MINOR GREATER 7)
 		NAMES jna
 		PATHS /usr/share/java
 		)
+	
+	if(NOT jna) # try fetching it, remember to update SHA1 if you update the jar
+		file(DOWNLOAD 
+			"https://maven.java.net/content/repositories/releases/net/java/dev/jna/jna/4.2.2/jna-4.2.2.jar" 
```

Comment:
I have no idea how they do it. The "clean" way I know is using maven but that doesn't seem to be a (clean) option here.